servicex/servicex_adapter.py

```diff
@@ -149,6 +148,6 @@ async def get_transform_status(self, request_id: str) -> TransformStatus:
                                 raise ValueError(f"Transform ID {request_id} not found")
                             o = await r.json()
                             return TransformStatus(**o)
-            except RuntimeError as e:
+            except RetryError as e:
```

Comment:
Added a RetryError exception handling instead of RuntimeError since that will capture the last retried error from the AsyncRetrying

Reply:
Not if we use `reraise=True`, no?

Reply:
For a short code block for the try catch, RuntimeError makes sense.  If the block was bigger we could have multiple exception handled.
Although I tested this using a small test block, both work fine. However, for now I will keep this RuntimeError exception.
I will revert the change.